libraries/Microsoft.Bot.Builder.LanguageGeneration/LGFileParser.g4

```diff
@@ -37,7 +37,8 @@ parameters
 
 templateBody
 	: normalTemplateBody						#normalBody
-	| conditionalTemplateBody					#conditionalBody
+	| ifElseTemplateBody					    #ifElseBody
+    | switchCaseTemplateBody                    #switchCaseBody
```

Comment:
format is not aligned

Reply:
checking the reason why it is not aligned here. It is actually aligned in my file.